payments-core/res/values-DA/strings.xml

```diff
@@ -61,58 +112,140 @@
+    <!-- Text for back button -->
     <string name="stripe_paymentsheet_back">Tilbage</string>
     <string name="stripe_paymentsheet_close">Luk</string>
+    <!-- Error when the user presses submit and they have not filled in the email field. -->
+    <string name="stripe_paymentsheet_error_email_blank">Din e-mail kan ikke være tom.</string>
+    <!-- Error when the user presses submit and they have not filled in the name field. -->
+    <string name="stripe_paymentsheet_error_name_blank">Dit navn kan ikke være tomt.</string>
+    <!-- label for text field to enter card expiry -->
     <string name="stripe_paymentsheet_expiration_date_hint">MM/ÅÅ</string>
+    <string name="stripe_paymentsheet_new_card_button_label">+ Nyt kort</string>
+    <!-- Title of a section containing a card form that will collect the payment information from the user. -->
     <string name="stripe_paymentsheet_or_pay_with_card">Eller betal med et kort</string>
+    <!-- Label of a button that initiates payment when tapped -->
     <string name="stripe_paymentsheet_pay_button_amount">Betal %s</string>
+    <!-- Label for a button that starts processing the payment. -->
     <string name="stripe_paymentsheet_pay_button_label">Betal</string>
+    <!-- Title for credit card number entry field -->
+    <string name="stripe_paymentsheet_payment_method_card">Kort</string>
+    <!-- Source type brand name -->
+    <string name="stripe_paymentsheet_payment_method_ideal">iDEAL</string>
+    <!-- Label of a button that, when tapped, initiates payment, becomes disabled, and displays this text -->
     <string name="stripe_paymentsheet_primary_button_processing">Behandler...</string>
+    <!-- The label of a switch indicating whether to save the user&apos;s card for future payments to this merchant -->
     <string name="stripe_paymentsheet_save_this_card_with_merchant_name">Gem dette kort til fremtidige %s betalinger</string>
+    <!-- Title shown above a carousel containing the customer&apos;s payment methods -->
     <string name="stripe_paymentsheet_select_payment_method">Vælg din betalingsmetode</string>
+    <!-- Label of a button displayed below a payment method form. Tapping the button sets the payment method up for future use -->
     <string name="stripe_paymentsheet_setup_button_label">Opsætning</string>
-    <string name="stripe_paymentsheet_total_amount">I alt: %s</string>
+    <!-- When there are multiple browsers and web authentication is required, this will show in a window asking which browser they want to use to "Verify your payment" -->
+    <string name="stripe_verify_your_payment">Bekræft din betaling</string>
+    <!-- An error message. -->
+    <string name="The_IBAN_you_entered_is_incomplete.">Det indtastede IBAN-nummer er ufuldstændigt.</string>
+    <!-- An error message. -->
+    <string name="The_IBAN_you_entered_is_invalid__is_not_a_supported_country_code.">Den indtastede IBAN er ugyldig, \"%s\" er ikke en gyldig landekode.</string>
```

Comment:
The strings that are not used in `payments-core` can be added only to `paymentsheet`. These (and others) are only used by the forms, `payments-core` doesn't need them.

Reply:
I considered this, but I think we should either just keep them in both, or only have them in the paymentsheet module.  I am not sure it is worth the effort splitting it out.   @brnunes-stripe

Reply:
Right, what I'm saying is have them only in the paymentsheet module. At least the new ones that are used only in the forms.